source/Engine.cpp

```diff
@@ -1625,8 +1625,22 @@ void Engine::HandleKeyboardInputs()
 	else
 		activeCommands.Clear(Command::WAIT);
 	
-	// Transfer all newly pressed, unhandled keys to active commands.
-	activeCommands |= keyDown;
+	// Translate shift+BACK to a command to a STOP command to stop all movement of the flagship.
```

Comment:
You use both "shift+BACK" and "BACK+SHIFT" in this comment. (Also, "autoPilot" is a variable name, while "autopilot" is a concept whose implementation in ES uses a variable named autoPilot.)